services/gateway/apis/http-version3/methods/transactions.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
 			max: 41,
 			pattern: regex.ADDRESS_LISK32,
 		},
+		receivingChainID: { optional: true, type: 'string', min: 1, pattern: regex.CHAIN_ID },
```

Comment:
Already covered by regex.

```suggestion
		receivingChainID: { optional: true, type: 'string', pattern: regex.CHAIN_ID },
```